### PR TITLE
feat: new vercel quickstart

### PR DESCRIPTION
**Pull request by jcountsNR**
# Summary

creating a new vercel quickstart.

## Pre merge checklist

- [ X] Did you check you NRQL syntax? - Does it work?
- [ X] Did you check your dashboard image quality? -  Do they look good?
- [ X] Did you include an InstallPlan and Documentation reference?
- [ X] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ X] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

**Comment by jcountsNR** [comment]
@aswanson-nr , I don't want to put too much pressure on this one, but I have a few customers already waiting to try this out. I double checked it and it looks good to me.

File: data-sources/vercel/config.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  We enable teams to iterate quickly and develop, preview, and ship delightful user experiences. Vercel has zero-configuration support for 35+ frontend frameworks and integrates with your headless content, commerce, or database of choice.
+
+icon: logo.svg
+install:
+  primary:
+    nerdlet:
+      nerdletId: nr1-install-newrelic.quickstart-installation-plan
+      nerdletState:
+        quickstartId:
```
Comment (aswanson-nr) [inline data-sources/vercel/config.yml:16]: We'll just need to get the generated vercel quickstart id and add it here

**Review by aswanson-nr — COMMENTED** [review]
Hey @jcountsNR, this looks pretty much good to go! 
For the data source, it's going to need to the quickstart id to tell the in-product experience which dashboards to install. So let's pull the data-source out of this PR and submit it in a secondary one once this quickstart gets merged and is assigned an id.

File: data-sources/vercel/config.yml (hunk 1 above)
Comment (jcountsNR) [reply data-sources/vercel/config.yml]: @aswanson-nr got it, I removed the data source so it should be good now. I'll add it in later.

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**